src/main/java/com/bina/varsim/types/stats/MapRatioRecordSum.java

```diff
@@ -1,25 +1,22 @@
 package com.bina.varsim.types.stats;
 
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
-import java.util.TreeMap;
+import java.util.*;
 
 /**
  * This maps variant type to a ratiorecordsum
  */
 public class MapRatioRecordSum {
 
     // use tree map so that the JSON is ordered...
-    private TreeMap<String, RatioRecordSum> data;
+    private EnumMap<EventTypesForStats, RatioRecordSum> data;
```

Comment:
Use `Map` instead of `EnumMap` here. `EnumMap` is the implementation and `Map` is the interface. We should use the most general interface possible.